### ISSUE TITLE
Config file description?

### ISSUE THREAD

**Issue by maihao14** [seisbench/pick-benchmark#7]
Hi @yetinam , Compared to tutorial training examples, this `training.py` is more comprehensive and helpful for training new models based on new data. When I try to rewrite the config JSON file, I meet some difficulties in rewriting my demand. 

For example, I saw the README.md mention user could use the local datasets in Seisbench format, but when I replace
  "data": "SCEDC" to   "data": "./seisbench/datasets/ethz"   (which is the path of the downloaded ETHZ dataset)  it will raise errors. 
Moreover, how to use `data.filter`  in JSON to change the complete built-in dataset? E.g., reduce the samples amount, filter magnitude range? Is it possible to work by rewrite the json file?  

Another question is could I load pre_trained weights in this `training.py`? I.E., how to express `sbm.PhaseNet.from_pretrained("stead")`  in json file (or rewrite some lines of `training.py`)? 

Tutorial `03a_training_phasenet.ipynb` is a really user-friendly notebook for training. However, it didn't contain the following part, e.g., export the trained model (weights, logging) to a local folder for later reused, generate performance matrix based on validation set, etc. [pick-benchmark](https://github.com/seisbench/pick-benchmark) have these scripts, so I'd like to create a notebook to extend  `03a_training_phasenet.ipynb`.  I would appreciate it if you could offer some advice.

**Comment by yetinam** [comment]
Hi @maihao14 ,

thanks for your questions. Regarding the documentation of the config parameters, there is some to be found in the docstrings but I admit it's not very complete. Unfortunately, I won't have the time to comprehensively document the parameters in the foreseeable future. I think most parameters should be rather easy to reverse-engineer though. A good starting point for this is [here](https://github.com/seisbench/pick-benchmark/blob/31057a1981f6cd50f828fbf8c284c2b0a228a502/benchmark/train.py#L31).

Regarding the dataset, it seems that the README is wrong. You'll have to do a code change to accomodate local datasets. The additional functionalities you mention, i.e., data filtering and pretrained weights are not built into the benchmark yet. Both are no major changes, but you'll need to modify the code to incorporate this.

Regarding the tutorial, `03a_training_phasenet.ipynb` is intended as a very minimalistic example. It would be great if you wanted to create a more extensive tutorial complementing it.
